protos/flyteidl/event/event.proto

```diff
@@ -151,12 +151,30 @@ message TaskExecutionEvent {
     // The version field should be incremented when metadata changes across the duration of an individual phase.
     uint32 phase_version                  = 12;    
 
+    // If there is an explanation for this phase transition, the reason will capture it.
+    string reason                         = 13;
+
+
+    // A predefined yet extensible Task type identifier. If the task definition is already registered in flyte admin
+    // this type will be identical, but not all task executions necessarily use pre-registered definitions and this
+    // type is useful to render the task in the UI, filter task executions, etc.
+    string type                           = 14;
```

Comment:
Is this the task type? or the event type? from the comment it seems it refers to the task type... maybe it should be renamed `task_type` ?

Reply:
ah good point, done